common/mccmapping.h

```diff
@@ -0,0 +1,23 @@
+/*
+    Copyright (C) 2022 Jolla Ltd.
+    Contact: Daniel Suni <daniel.suni@jolla.com>
+
+    This file is part of geoclue-mlsdb.
+
+    Geoclue-mlsdb is free software; you can redistribute it and/or
+    modify it under the terms of the GNU Lesser General Public
+    License as published by the Free Software Foundation; either
+    version 2.1 of the License.
+*/
+
+#ifndef GEOCLUE_MCC_MAPPING
+#define GEOCLUE_MCC_MAPPING
+
+// Mcc numbers go up to 750, which requires 10 bits to store. There are,
+// however only 222 unique instances of them. This only requires 8 bits to
+// store, and as it happens this allows us to cram the 66 bits of data needed
+// for the unique cell id into a 64-bit integer.
+// The 1024 at the end serves to prevent array overruns.
+short mccMap[] = {202, 204, 206, 208, 212, 213, 214, 216, 218, 219, 220, 221, 222, 226, 228, 230, 231, 232, 234, 235, 238, 240, 242, 244, 246, 247, 248, 250, 255, 257, 259, 260, 262, 266, 268, 270, 272, 274, 276, 278, 280, 282, 283, 284, 286, 288, 289, 290, 292, 293, 294, 295, 297, 302, 308, 310, 311, 312, 313, 314, 315, 316, 330, 334, 338, 340, 342, 344, 346, 348, 350, 352, 354, 356, 358, 360, 362, 363, 364, 365, 366, 368, 370, 372, 374, 376, 400, 401, 402, 404, 405, 410, 412, 413, 414, 415, 416, 417, 418, 419, 420, 421, 422, 424, 425, 426, 427, 428, 429, 430, 431, 432, 434, 436, 437, 438, 440, 441, 450, 452, 454, 455, 456, 457, 460, 466, 467, 470, 472, 502, 505, 510, 514, 515, 520, 525, 528, 530, 537, 540, 541, 542, 543, 546, 547, 549, 602, 603, 604, 605, 606, 607, 608, 609, 610, 611, 612, 613, 614, 615, 616, 617, 618, 619, 620, 621, 622, 623, 624, 625, 626, 627, 628, 629, 630, 631, 632, 633, 634, 635, 636, 637, 638, 639, 640, 641, 642, 643, 645, 646, 647, 648, 649, 650, 651, 652, 653, 654, 655, 657, 658, 659, 702, 704, 706, 708, 710, 712, 714, 716, 722, 724, 730, 732, 734, 736, 738, 740, 744, 746, 748, 750, 1024};
```

Comment:
Suppose this might need to be updated at some point. How should that happen? Supposedly together with the MCC map in tool_wrapper.sh?

Could have some comment here to help the next guy. Or pondering would it make sense to have some tool to generate the array, having only one instance of the mapping to be manually updated.